docs/Synonyms.md

```diff
@@ -18,12 +18,16 @@ The definition of the synonym is the same as the primary definition, but has add
 
 For example, pod is a narrow synonym of fruit. 
 
+**Note** - when adding a narrow synonym, please first consider whether a new subclass should be added instead of a narrow synonym. If there is any uncertainty, start a discussion on the GitHub issue tracker.
+
 #### Broad
 
 The primary definition accurately describes the synonym, but the definition of the synonym may encompass other structures as well. In some cases where a broad synonym is given, it will be a broad synonym for more than one ontology term.
 
 For example, Cyst of eyelid has the broad synonym Lesion of the eyelid.
 
+**Note** - when adding a narrow synonym, please first consider whether a new superclass should be added instead of a broad synonym. If there is any uncertainty, start a discussion on the GitHub issue tracker.
```

Comment:
I think you mean "when adding a broad synonym" here

Reply:
yes I do!, I'll fix